### PR TITLE
feat(ui): Skip empty elements in `Stack` and `Inline` components.

### PR DESCRIPTION
**Pull request by doniyor2109**

### CODE REVIEW

**Comment by umidbekk** [comment]
@doniyor2109 can you provide examples on storybook with the screenshots?

**Comment by umidbekk** [comment]
and maybe it's better to hide empty elements completelty?

**Comment by doniyor2109** [comment]
> and maybe it's better to hide empty elements completelty?

Hmm. You mean by `display:none` ?

**Comment by umidbekk** [comment]
> > and maybe it's better to hide empty elements completelty?
> 
> Hmm. You mean by `display:none` ?

yes

File: packages/ui/src/inline/Inline.stories.tsx [hunk 1]
```diff
@@ -31,6 +31,8 @@ export default {
   },
 } as Meta;
 
+const Empty = () => null;
```
Comment (umidbekk) [inline packages/ui/src/inline/Inline.stories.tsx:34]: this will fail in Playroom example, you can add it to the `__docs__`

**Review by umidbekk — CHANGES_REQUESTED** [review]

**Comment by doniyor2109** [comment]
> and maybe it's better to hide empty elements completely?

I have tried with `display: none` but there was some challenge with `&:first-child {padding-top: 0;}`. If empty element is first child then second element is adding extra padding (`first-child` rule not applying). 

Let me know if you have solution for this case.

**Comment by umidbekk** [comment]
> > and maybe it's better to hide empty elements completely?
> 
> I have tried with `display: none` but there was some challenge with `&:first-child {padding-top: 0;}`. If empty element is first child then second element is adding extra padding (`first-child` rule not applying).
> 
> Let me know if you have solution for this case.

what about that?

```css
&:empty {
  display: none;
}

&:not(:empty):first-child {
  padding-top: 0;
}

```

**Comment by umidbekk** [comment]
@doniyor2109 it didn't work, but I could achieve this with the `~` https://jsfiddle.net/2kbz51ty/20/

**Comment by umidbekk** [comment]
@doniyor2109 updated: https://jsfiddle.net/2kbz51ty/31/

here is the css

```css
&:empty {
  display: none;
}
  
&:not(:empty) ~ & {
 padding-top: 15px;
}
```

How it works:
1. Hides all the empty elements
2. Finds non empty elements, and sets the padding for all the siblings that goes **after** the current element

**Comment by doniyor2109** [comment]
@umidbekk Still `Empty` is not found in playroom(
https://preview-170--ui-superdispatch.netlify.app/?path=/playroom/layout-inline--empty-elements

**Comment by umidbekk** [comment]
@doniyor2109 oh, you also need to export it from the https://github.com/superdispatch/ui/blob/55c498d2ec82a08e0b82a9a70201937e380258db/scripts/generate-playroom-components.ts#L22

**Pull request merged**